### DISCUSSION TITLE
Aqara Dual relay module T2 LLKZMK12LM unknown key 65522

### DISCUSSION THREAD

**Discussion by tibinoo** [Koenkk/zigbee2mqtt#21594] [Devices]
Hi, i have more error in my log.
10:44:56LLKZMK12LM: unknown key 65522 with value �u�:A
What is it?

**Comment by half-a-yoyo** [comment]
Hi all,
same problem here:

> [2025-09-22 13:52:32] [34mdebug[39m: 	z2m: Received Zigbee message from 'Aqara outlet H2 EU (Kue 3er mi)', type 'attributeReport', cluster 'manuSpecificLumi', data '{"65522":{"data":[170,117,19,132,160,170,7,65,0,15,12,35,1,1,1,9,10,35,8,0,12,100,11,32,2],"type":"Buffer"}}' from endpoint 1 with groupID 0
> [2025-09-22 13:52:32] [34mdebug[39m: 	zhc:lumi: WP-P01D: unknown key 65522 with value �u���A�#	
> #�d 


My Aqara Smart wall outlet ( WP-P01D -> https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/WP-P01D.md ) is flooding my protocols with this <s>error</s> message.

It's related to endpoint 1. Either `genOta` or `genTime` but I'd _guess_ it's genTime because `Last seen` is `N/A`. 🤔 

A few days later (I bed you can _hear_ the sentence! 😆 )
AHA! Found the "bad guy"!

It _is_ last_seen! You may add following to your `/homeassistant/zigbee2mqtt/configuration.yaml`

```
advanced:
  last_seen: ISO_8601
```

The `advanced:` should be present already. Add the line `last_seen: ISO_8601` (pay attention to the correct indentation!) and restart HA. :wink:
